### PR TITLE
K8s tests fix

### PR DESCRIPTION
**Pull request by hunchback**
Fixed k8s tests (for checking creation of nodes). Please note that one has to run `scripts/ci/install-minikube.sh` on the target test VM prior to running the tests.

### CODE REVIEW

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by hunchback** [comment]
It appears minikube not running on test VMs:

```
10:28:53.540 2018-02-12T11:28:53.473+0100	INFO	http/server.go:134 (*Server).Listen	dpdk3: Listening on TCP socket 0.0.0.0:8082
10:28:53.547 E0212 11:28:53.479981   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1.Pod: Get https://127.0.0.1:8443/api/v1/pods?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
10:28:53.549 E0212 11:28:53.480825   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1beta1.NetworkPolicy: Get https://127.0.0.1:8443/apis/extensions/v1beta1/networkpolicies?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
10:28:53.682 E0212 11:28:53.480910   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1.Pod: Get https://127.0.0.1:8443/api/v1/pods?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
10:28:53.682 E0212 11:28:53.481516   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1.Pod: Get https://127.0.0.1:8443/api/v1/pods?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
10:28:53.682 E0212 11:28:53.481720   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1.Node: Get https://127.0.0.1:8443/api/v1/nodes?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
10:28:53.683 E0212 11:28:53.481845   18325 reflector.go:205] github.com/skydive-project/skydive/topology/probes/k8s/client.go:141: Failed to list *v1.Namespace: Get https://127.0.0.1:8443/api/v1/namespaces?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
```

File: tests/k8s_test.go [hunk 1]
```diff
@@ -29,72 +29,81 @@ import (
...
-		{"kubectl create -f " + networkPolicyConfig, true},
-	}
-	tearDownNetworkPolicy = []helper.Cmd{
-		{"kubectl delete -f " + networkPolicyConfig, false},
+}
+
+func tearDownFromDeployment(name string) []helper.Cmd {
+	return []helper.Cmd{
+		{"kubectl delete deployment " + k8sObjectName(name), false},
+		{"sleep 2", true},
```
Comment (lebauce) [inline tests/k8s_test.go]: I would prefer if we tested that the deployment/pod/networkpolicy was successfully deleted instead of waiting 2 seconds :-/

Comment (hunchback) [reply tests/k8s_test.go]: ok, have addressed

Comment (lebauce) [reply tests/k8s_test.go]: I was suggesting to check that the resource was successfully removed from the graph. That's ok, we'll address this in an other PR

**Pull request merged**